app/views/posts/show.html.erb

```diff
@@ -1,3 +1,13 @@
-<h1>Here is a page for showing a post in detail and its comments</h1>
-
-<p>This page is under construction</p>
+<div class="user_con">
+<div class="user_data_post"> 
+  <h2><%= @post.title %>by <span><%= @post.user.name %></span></h2>
```

Comment:
- [ ] Kindly make sure that styles  don't break while the quantity of comments increase: 
<img width="1377" alt="Screenshot 2022-05-03 at 12 52 57" src="https://user-images.githubusercontent.com/77038610/166427264-a15a2d09-4e84-43b2-8d69-8bd951643114.png">